test/node/src/insert.test.ts

```diff
@@ -742,20 +742,20 @@ for (const dialect of DIALECTS) {
 
       testSql(query, dialect, {
         postgres: {
-          sql: 'insert into "person" ("first_name", "last_name", "middle_name", "gender") values ($1, $2, $3, $4), ($5, default, default, $6)',
-          parameters: ['Foo', 'Spam', 'Bo', 'other', 'Baz', 'other'],
+          sql: 'insert into "person" ("first_name", "last_name", "middle_name", "gender") values ($1, $2, default, $3), ($4, default, default, $5)',
+          parameters: ['Foo', 'Spam', 'other', 'Baz', 'other'],
         },
         mysql: {
-          sql: 'insert into `person` (`first_name`, `last_name`, `middle_name`, `gender`) values (?, ?, ?, ?), (?, default, default, ?)',
-          parameters: ['Foo', 'Spam', 'Bo', 'other', 'Baz', 'other'],
+          sql: 'insert into `person` (`first_name`, `last_name`, `middle_name`, `gender`) values (?, ?, default, ?), (?, default, default, ?)',
+          parameters: ['Foo', 'Spam', 'other', 'Baz', 'other'],
         },
         mssql: {
-          sql: 'insert into "person" ("first_name", "last_name", "middle_name", "gender") values (@1, @2, @3, @4), (@5, default, default, @6)',
-          parameters: ['Foo', 'Spam', 'Bo', 'other', 'Baz', 'other'],
+          sql: 'insert into "person" ("first_name", "last_name", "middle_name", "gender") values (@1, @2, default, @3), (@4, default, default, @5)',
+          parameters: ['Foo', 'Spam', 'other', 'Baz', 'other'],
         },
         sqlite: {
-          sql: 'insert into "person" ("first_name", "last_name", "middle_name", "gender") values (?, ?, ?, ?), (?, null, null, ?)',
-          parameters: ['Foo', 'Spam', 'Bo', 'other', 'Baz', 'other'],
+          sql: 'insert into "person" ("first_name", "last_name", "middle_name", "gender") values (?, ?, null, ?), (?, null, null, ?)',
+          parameters: ['Foo', 'Spam', 'other', 'Baz', 'other'],
         },
```

Comment:
should probably make a separate test case for this.

Reply:
It felt redundant to me as there's another test case that already covers this as far as I remember

Reply:
Won't be able to continue this PR until next week as I don't have my laptop on me, let's talk over Discord and decide what to do